openedx/core/djangoapps/content/search/tests/test_documents.py

```diff
@@ -477,3 +477,112 @@ def test_collection_with_published_library(self):
                 "num_children": 1
             }
         }
+
+    def test_mathjax_plain_text_conversion_for_search(self):
+        """
+        Test how an HTML block with mathjax equations gets converted to plain text in search description.
+        """
+        # pylint: disable=line-too-long
+        eqns = [
+            # (input, expected output)
+            ('Simple addition: \\( 2 + 3 \\)', 'Simple addition:  2 + 3'),
```

Comment:
Should we be replacing these if they do not have `[mathjaxinline]` `[/mathjaxinline]` around them? I would think we only want to replace explicit MathJax areas.

Reply:
Mathjax equations inside `\( <eqn> \)` and `\[ <eqn> \]`) is also rendered in html and problem blocks.

Reply:
I see :)